sdk/go/common/apitype/updates.go

```diff
@@ -70,6 +70,13 @@ type UpdateMetadata struct {
 	Environment map[string]string `json:"environment"`
 }
 
+// Message is a message from the backend to be displayed to the user.
+type Message struct {
+	Message       string `json:"message"`
+	Type          string `json:"type,omitempty"`
+	PriorityOrder int    `json:"priorityOrder,omitempty"`
```

Comment:
This was in our notes document, I'm not sure if we need these yet. I think it'd be best to wait until the service has something that they want to do before adding these.